go/pkg/client/cas.go

```diff
@@ -395,6 +395,15 @@ func (c *Client) readBlobStreamed(ctx context.Context, hash string, sizeBytes, o
 	return n, nil
 }
 
+// ReadProto reads a blob from the CAS and unmarshals it into the given message.
+func (c *Client) ReadProto(ctx context.Context, d digest.Digest, msg proto.Message) error {
```

Comment:
Thanks for adding this! Could you please change line 534 here in FlattenActionOutputs to use this function instead? It will save a few lines.

Reply:
Sure, have done.

I've changed it slightly; looks like before it was silently ignoring errors on reading that blob, which seemed wrong? Let me know if I've misunderstood though...

Reply:
No, you're right, that was weird! Thank you!